conf/sip-communicator.properties

```diff
@@ -0,0 +1,2 @@
+org.jitsi.impl.neomedia.transform.srtp.SRTPCryptoContext.checkReplay=false
+org.jitsi.videobridge.TCP_HARVESTER_PORT=4443
```

Comment:
In the install script you have :
```bash
port=$(ynh_find_port --port=4443)
```

You should replace the port with \_\_PORT\_\_ and use `ynh_replace_string`

Reply:
Fixed :)